src/view/components/toolbar/InputSlider.tsx

```diff
@@ -31,8 +34,9 @@ class InputSlider extends React.Component<ISliderProps, any, any> {
                     onInput={this.handleOnChange}
                     defaultValue={this.props.minValue.toLocaleString()}
                     pattern={`^-?[0-9]{0,${
-                        this.props.maxValue.toString().length
-                    }}$`}
+                        (this.props.maxValue / this.props.step).toString()
```

Comment:
What is this pattern for?

Reply:
It's to control the input of the user for sensor value. to limit the number of digit after the period also to limit the max value by limiting length of input.